x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/source_settings.tsx

```diff
@@ -172,9 +174,9 @@ export const SourceSettings: React.FC = () => {
             baseUrl={baseUrl}
           />
           <EuiFormRow>
-            <Link to={editPath}>
-              <EuiButtonEmpty flush="left">{SOURCE_CONFIG_LINK}</EuiButtonEmpty>
-            </Link>
+            <EuiButtonEmptyTo to={editPath} flush="left">
+              {SOURCE_CONFIG_LINK}
+            </EuiButtonEmptyTo>
```

Comment:
@scottybollinger / @yakhinvadim - I recommend taking a look at this commit (https://github.com/elastic/kibana/pull/104792/commits/50e49a483d4a9a1cf838b8316eecd98227c36914) and doing a QA pass across your app for 2 things:

- Remove any old references to `<Link>` and replacing them with our `Eui*To` React Router helpers
- Checking that **all** your links/SPA navigations work in dev mode with a `/xyz` basepath. Please note that the intent of the dev basepath is to check that your app still works for customers who have their Kibana set to non-root URLs, e.g. `somesite.com/kibana`. If you don't QA for this and occasionally run in dev mode, you will at some point get SDH issues or reports in the future complaining that certain links aren't working on their install

Reply:
Thanks for the heads up. Will do. Just doing a quick pass shows that this was the only `Link` so great catch and thanks for fixing